books/free-programming-books-pt_BR.md

```diff
@@ -166,6 +166,7 @@
 #### Arquitetura de Software
 
 * [ASP.NET Core architecture](https://docs.microsoft.com/pt-br/dotnet/architecture/modern-web-apps-azure/) (PDF)
+* [Melhores Práticas de Arquitetura de Software na era da Nuvem](https://leanpub.com/manual-arquitetura-software) (Necessário criar uma conta (gratuita) no Leanpub para baixar o livro completo nos formatos PDF, EPUB, MOBI ou pelo próprio site)
```

Comment:
I don't think this comment after the link is necessary.

Reply:
I agree, I did it just to follow the standard.